examples/design/EU-DEMO/EUDEMO_builders/ivc/ivc_boundary.py

```diff
@@ -0,0 +1,81 @@
+# bluemira is an integrated inter-disciplinary design tool for future fusion
+# reactors. It incorporates several modules, some of which rely on other
+# codes, to carry out a range of typical conceptual fusion reactor design
+# activities.
+#
+# Copyright (C) 2021 M. Coleman, J. Cook, F. Franza, I.A. Maione, S. McIntosh, J. Morris,
+#                    D. Short
+#
+# bluemira is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# bluemira is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with bluemira; if not, see <https://www.gnu.org/licenses/>.
+"""
+IVC Boundary Designer
+"""
+from dataclasses import dataclass
+from typing import Dict
+
+from bluemira.base.designer import Designer
+from bluemira.base.error import DesignError
+from bluemira.base.parameter_frame import NewParameter as Parameter
+from bluemira.base.parameter_frame import NewParameterFrame as ParameterFrame
+from bluemira.builders.tools import varied_offset
+from bluemira.geometry.wire import BluemiraWire
+
+
+@dataclass
+class IVCBoundaryParams(ParameterFrame):
+    """Parameters for running the `WallSolver`."""
+
+    tk_bb_ib: Parameter[float]  # Inboard blanket thickness
+    tk_bb_ob: Parameter[float]  # Outboard blanket thickness
+    ib_offset_angle: Parameter[float]  # 45 degrees
+    ob_offset_angle: Parameter[float]  # 175 degress
```

Comment:
Should we consider having default values in some of these parameter frames at some point? Or is that something we're trying to avoid?

Reply:
I think we're trying to avoid them. It was a comment so the old default wasn't lost (although it is in the test). We could also have a conversation about people wanting defaults in their individual reactors